JS/functions.js

```diff
@@ -0,0 +1,390 @@
+*/
+
+//returns 10
+const ten = () =>{
+	return 10;
+}
+
+console.log(ten());
+
+
+// A return is no way the same as a console.log
+// A function only has a value if it returns something
+
+
+
+const newMultiply = (n1,n2)=>{
+
+	return n1 * n2;
+}
+
+let a = 3, b = 3;
+console.log('a times b '+ newMultiply(a,b));
+
+// you can use a function as its value
+// you can use a called function to represent an argument being passed into a function
+
+console.log(newMultiply(2, newMultiply(a,b)));
+
+
+
+const example =(input)=>{
+	if(input === 'none'){
+		return 0;
+	}
+	return 1;
+}
+//prints 0
+console.log(example('none'));
+
+
+
+const calculateArea = (L,W)=>{
+	return L * W;
+}
+
+const area = calculateArea(2,2);
+
+console.log(area);
+
+//sums up three numbers and converts to string
+const stringify = (a,b,c)=>{
+	let total = a+b+c;
+	return total.toString();
+}
+
+console.log(stringify(1,2,3));
+console.log(typeof(stringify(1,2,3)));
+
+
+
+
+const checkAreIdentical = (a,b)=>{
+	return a === b;
+}
+console.log(checkAreIdentical(5,5));
+
+
+
+//Lab
+//1
+const printGreeting = (name)=>{
+	return "Hello there " + name;
+}
+
+console.log(printGreeting('Joe'));
+
+//2
+const reverseWorder = (str)=>{
+	let letters="";
+	for (let i = 1; i <= str.length; i++){
+		letters += str[str.length-[i]];
+	}
+	return letters
+}
+
+console.log(reverseWorder("Hello Taylor"));
+
+//3
+const calculate = (num1,num2,operation)=> {
+	if (operation === "add"){
+		return num1 + num2;
+	}
+	else if(operation === "subtract"){
+		return num1 - num2;
+	}
+	else if(operation === "div"){
+		return num1/num2;
+	}
+	else if(operation === "mult"){
+		return num1 * num2;
+	}
+	else if(operation === "exp"){
+		return Math.pow(num1,num2);
+	}
+}
+
+console.log(calculate(2,3,"exp"));
+
+
+//4
+
+
+const reverseWordOrder = (str)=>{
+	let reverseString = '';
+	let removeSpace = str.split(' ');
+	for(let i = 1; i <= removeSpace.length; i++){
+		reverseString += removeSpace[removeSpace.length - [i]] + ' ';
+
+	}
+	return reverseString;
+}
+console.log(reverseWordOrder('My name is Johnny'));
+
+//5
+
+const checkPandigital = (num)=> {
+	
+	const len = num.toString().length;
+	const digitsToString = num.toString().split('');
+
+	for(let i = 0; i < len; i ++){
+		if(digitsToString[i] <= 0){
+			return "Zero or below is detected";
+		}
+		else if(digitsToString[i] > len){
+			return "Not Pandigital "+ digits[i]+ " is greater than "+ len; 
+		}
+		else{
+			for(let j = i+1; j < len; j ++){
+				if(digitsToString[j] === digitsToString[i]){
+					return "Not Pandigital";
+				}
+			}
+		}
+
+	}
+	
+	return "Number is Pandigital";
+
+}
+
+console.log(checkPandigital(123453));
+
+
+//6
+
+const checkPalindrome = (word)=>{
+	let flipper = "";
+	for(let i = 1; i <= word.length; i ++){
+		flipper += word[word.length - [i]];
+	}
+	if(flipper === word){
+		return true;
+	}
+	else{return false;}
+}
+
+
+console.log(checkPalindrome('racecar'));
```

Comment:
everything above this line is from a separate assignment?  
please submit only the assigned questions in homework responses/use a separate repo

Reply:
Hw Complete. In new Js File named this_is_the_hw.js

Reply:
Please mark in Garnet